job-descriptions/product-manager.md

```diff
@@ -0,0 +1,74 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Product Manager
+
+### About us
+
+We're building the new standard developer platform. Top tech companies have invested \$100Ms to build internal developer platforms for code search, code review, alerts, and automation. Sourcegraph provides this standard developer platform to every company, helping startups and large enterprises ship better software faster.
+
+[Our mission](https://sourcegraph.com/plan) is to dramatically increase the number of people who can understand and write code. By making code more accessible, we will democratize software development and accelerate innovations that bring the future sooner in transportation, health care, energy, AI, communication, space travel, etc.
+
+We value openness and transparency. [Our handbook](https://about.sourcegraph.com/handbook), [our strategy](https://about.sourcegraph.com/company/strategy), and [our code](https://github.com/sourcegraph/sourcegraph) are public. This transparency helps us rapidly gather feedback from our customers so we can iterate, learn, and deliver the best product.
+
+To create a product that serves the needs of all developers, we are building a diverse remote-first team that is distributed across the world. Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
+
+You'll work with developers and other team members (from sales, marketing, and design) to plan, ship, and grow specific areas of our product. As an early member of our product team, you'll have significant product ownership and will communicate directly with our most sophisticated and demanding users to help preserve our market leadership.
+
+Why this role is special:
+
+- Our product has [extremely high engagement](https://engineeringblog.yelp.com/2019/11/winning-the-hackathon-with-sourcegraph.html) relative to most products (especially enterprise products). We get enormous amounts of user feedback from our unusually thoughtful user base.
+- Our developers use the product, so they care deeply about product matters. You won't be fighting apathy.
+
+Our [public roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit) shows the product priorities for each engineering team and we use [OKRs](https://about.sourcegraph.com/company/okrs) to communicate company goals for each quarter.
+
+#### Responsibilities
+
+You will:
+
+- Own an area of the product, and create and drive its product roadmap.
+- Collaborate with engineers and designers to ideate, plan, and build the solutions to customer problems.
```

Comment:
I have a negative reaction to that word "ideate"; it seems very corporate to me. Can we avoid it or use a different word?

Reply:
Is brainstorm better?

Reply:
Design?

Reply:
Conceptualize?

Reply:
Brainstorm is fine. What if we just left it as "plan and build"? or "create, plan, and build"

```suggestion
- Collaborate with engineers and designers to plan and build the solutions to customer problems.
```